src/sentry/south_migrations/0365_backfill_first_project_heroku.py

```diff
@@ -0,0 +1,1001 @@
+# -*- coding: utf-8 -*-
+from sentry.models.organizationonboardingtask import OnboardingTask, OnboardingTaskStatus
```

Comment:
We can’t safely import these here. We should use the actual values these relate to inside the migration.

Reply:
Why?

Reply:
Migrations need to work perpetually into the future. It’s the same reason you pull the models out of the `orm` object. 

Imagine 3 months from now, say this feature were deleted. Or renamed, or moved, etc. this migration still runs on a fresh install. So if it can’t find the import, it means we’ve broken Sentry for any new install until we come back and edit this migration.

It’s easier to think of migrations as a point in time snapshot, and you should materialize whatever you need for this migration to work into the migration itself. That was as things move around for implementation detail, we don’t break migrations inadvertently.

Another alternative scenario is that the value of these changes in the future. Now it does the wrong thing in the migration.